lib/yaml.dart

```diff
@@ -2,6 +2,7 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
+import 'dart:io';
```

Comment:
I think this is unwise as it adds a dependency on `dart:io`, which means that `pub.dev` will classify the package as not working on web.